packages/babel-plugin-remove-graphql-queries/src/index.js

```diff
@@ -168,6 +168,23 @@ function isUseStaticQuery(path) {
   )
 }
 
+function getGraphqlExpression(t, queryHash, source) {
```

Comment:
We are creating object here, so `get` "operation" threw me a bit at first. Could this have more descriptive naming? Maybe something just replace `get` with `create` or `generate`.

Also some description what this is doing (jsdoc) would be nice.

(This is preemptive comment so I don't forget and doesn't need to be addressed quickly - we still want to evaluate if replacing current query hash with object is the way to go)

Reply:
Hello @pieh,
I've made the changes and added some jsdoc style code-comments in commit 
[11a6f77](https://github.com/gatsbyjs/gatsby/pull/23423/commits/11a6f774fa28c42c605c6b955d8baa2dbc63120a)